RecoLocalCalo/EcalRecProducers/plugins/AmplitudeComputationCommonKernels.cu

```diff
@@ -247,8 +263,8 @@ namespace ecal {
           // likely false
           if (check_hasSwitchToGain0) {
             // assign for the case some sample having gainId == 0
-            //energies[ch] = amplitudes[ch][sample_max];
-            energies[ch] = amplitude;
+            //energies[inputCh] = amplitudes[ch][sample_max];
+            energies[inputCh] = amplitude;
```

Comment:
why the change from `ch` to `inputCh` ?
what is the difference ?

Reply:
previously there was a single ptr for the output collections (eb was first and then ee). now i switched to 2 ptrs, one for each collection given that each collection has its own unique_ptr that manages memory. `inputCh` is computed having this in mind... ch is just a linear idx.

Reply:
understood, thanks